### PR TITLE
Aegis 244

### PR DESCRIPTION
**Pull request by TanujaG**
@loahou04 @richweclg @marcdonovan @vshivp2

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -156,9 +167,11 @@ class NotificationComponent {
 			render: function() {
 				return (
 					<div>
+						<IntlProvider {..._this.intlData}>
```
Comment (loahou04) [inline main.js]: I think this is suppose to be the base.  You don't need the div wrapping it as well

**Comment by loahou04** [comment]
approved

File: src/js/NotificationBlankState.js [hunk 2]
```diff
@@ -2,6 +2,37 @@ import React from 'react';
...
+		id: 'errorMessage.description',
+		defaultMessage: 'There seems to be a problem with this feature.  Try refreshing your browser or clearing your cache.'
+	},
+	notificationBlankStateHeading: {
+		id: 'notificationBlankState.heading',
+		defaultMessage: 'Nothing yet!'
+	},
+	notificationsBlankStateDescription: {
+		id: 'notificationBlankState.description',
+		defaultMessage: 'We’ll let you know when something comes up. Till then, browse your'
```
Comment (marcdonovan) [inline src/js/NotificationBlankState.js]: browse my what?

File: translations/zh.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "errorMessage.heading": "系统遇到问题。",
+	"errorMessage.description": "这个功能有错误。请尝试刷新浏览器或清除缓存。",
+	"notificationBlankState.heading": "目前无任何内容！",
+	"notificationBlankState.description": "有新内容时，我们会通知您。",
+	"notificationBlankState.link":"查看推送记录。",
+	"archiveBlankState.heading": "目前无任何推送！",
+	"archiveBlankState.description": "这是推送记录页面",
+	"notification.heading":"通知",
+	"backToNotification.heading":"前往推送记录",
+	"backToArchive.heading":"Back to Notifications Archive",
```
Comment (marcdonovan) [inline translations/zh.json]: oops. I don't speak Mandarin, but this is def not that.

**Comment by marcdonovan** [comment]
approve

**Pull request merged**